SPECS/dracut/dracut.spec

```diff
@@ -75,6 +79,20 @@ Requires:       nss
 This package requires everything which is needed to build an
 initramfs with dracut, which does an integrity check.
 
+%package hostonly
```

Comment:
Isn't there an implied linkage between these dracut subpackages and whether the kernel supplies the underlying modules? Should each kernels "Requires" their relevant dracut subpackages?

Reply:
A given kernel doesn't require any module to go in initramfs. A given image being deployed on a certain **platform** is what defines the need for certain modules. Hence, we are choosing to compose the set of dracut packages in the image at the imageconfig level.

Reply:
Makes sense about platforms defining what modules are in the initramfs. As we were chatting offline and you should consider if a "Requires: dracut >= X" in the kernel spec would help protect against regressions due to kernel module changes in kernel package updates